styleguide/derek/global-components/footer/footer.ejs

```diff
@@ -1,54 +1,72 @@
+        <ul class="footer-menu">
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Rackspace Blog</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Rackspace Newsroom</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Solve: Thought Leadership</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Technical Blog</a></li>
+        </ul>
+      </div>
 
-    <div id="footer-tab-4" class="col-sm-2">
-      <ul class="footer-menu">
-        <li class="footer-menu-item"><a class="footer-menu-link" href="#">Privacy Statement</a></li>
-        <li class="footer-menu-item"><a class="footer-menu-link" href="#">Website Terms</a></li>
-        <li class="footer-menu-item"><a class="footer-menu-link" href="#">Trademarks</a></li>
-        <li class="footer-menu-item"><a class="footer-menu-link" href="#">Sitemap</a></li>
-        <li class="footer-menu-item text-nowrap"><a class="footer-menu-link" href="#">UK Modern Slavery Statement</a></li>
-      </ul>
+      <div id="footer-tab-3" class="col-sm-3">
+        <ul class="footer-menu">
+          <li class="footer-menu-item"><a class="footer-menu-link extLink extLink-small" href="#">Support Overview</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Developers</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link extLink extLink-small" href="#">Support Documentation</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link extLink extLink-small" href="#">System Status</a></li>
+        </ul>
+      </div>
+
+      <div id="footer-tab-4" class="col-sm-3">
+        <ul class="footer-menu">
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Legal Overview</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Privacy Center</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Trademarks</a></li>
+          <li class="footer-menu-item"><a class="footer-menu-link" href="#">Website Terms</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="col-sm-3">
+      <div class="footer-social">
+        <a href="https://www.facebook.com/rackspace" target="_blank" class="basement-iconListItemLink">
+          <span class="fa fa-2x fa-facebook-official"></span>
+        </a>
+        <a href="https://twitter.com/Rackspace" target="_blank" class="basement-iconListItemLink">
+          <span class="fa fa-2x fa-twitter"></span>
+        </a>
+        <a href="https://www.linkedin.com/company/rackspace/" target="_blank" class="basement-iconListItemLink">
+          <span class="fa fa-2x fa-linkedin"></span>
+        </a>
+        <a href="https://www.youtube.com/rackspace" target="_blank" class="basement-iconListItemLink">
+          <span class="fa fa-2x fa-youtube"></span>
+        </a>
```

Comment:
For accessibility purposes, we'll need text inside the `<span>`.  You can then use CSS to hide that text, but we need it for screen readers. 

So for example: 
```<span class="fa fa-2x fa-facebook-official">Rackspace on Facebook</span>```